### PR TITLE
Added PlayerItemBreakEvent; Adds BUKKIT-1600

### PR DESCRIPTION
**Pull request by turt2live**
Jira: https://bukkit.atlassian.net/browse/BUKKIT-1600

CraftBukkit PR: https://github.com/Bukkit/CraftBukkit/pull/762

### CODE REVIEW

**Comment by sabersamus** [comment]
You do know, BlockBreakEvent contains a player object, right?

nvm: i get what this is now :P

**Comment by turt2live** [comment]
This is Item Break, like when your shovel snaps on dirt, not when you break the dirt

File: src/main/java/org/bukkit/event/player/PlayerItemBreakEvent.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.bukkit.event.player;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * Fired when a player's item breaks upon breaking a block
```
Comment (jrtc27) [inline src/main/java/org/bukkit/event/player/PlayerItemBreakEvent.java]: Should it not include interacting e.g. Flint and Steel?

Comment (turt2live) [inline src/main/java/org/bukkit/event/player/PlayerItemBreakEvent.java]: The leaky only wanted hand tools, so it only deals with hand tools
On May 7, 2012 3:22 PM, "James Clarke" <
reply@reply.github.com>
wrote:

Comment (mbax) [inline src/main/java/org/bukkit/event/player/PlayerItemBreakEvent.java]: Why limit yourself to that when you could cover all items breaking?

Comment (turt2live) [inline src/main/java/org/bukkit/event/player/PlayerItemBreakEvent.java]: I'll maybe rebase it later. As it kinda makes more sense that way.

**Comment by amaranth** [comment]
Pulled, thanks.

**Pull request closed without merging**